### PR TITLE
New rules

### PR DESCRIPTION
**Pull request by LandonSchropp**
I've noticed that we're a bit inconsistent in our function arguments style. I've seen all of the following in our code:

```
foo(a, b, c);

foo(
  a,
  b,
  c
)

foo(a,
  b,
  c
)

foo(a,
  b, c)
```

I'd like to simplify this and keep things consistent. I propose we stick with the first two styles—the first when the line is short enough and the second when it's not. There's now a simple ESLint rule for this, so this should simplify things for anyone with an editor that auto formats.

This PR also fixes the publish script, which has had issues in the past.

### CODE REVIEW

File: bin/publish [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Add the NPM token to the .npmrc file
+echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> "$HOME/.npmrc"
```
Comment (jorge0136) [inline bin/publish]: Do we already have this token as an env var within codeship?

File: bin/publish [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Publish the package, but only if the current published version is behind the latest version.
+PUBLISHED_VERSION=$(yarn info --json | jq -r '.data."dist-tags".latest')
+PACKAGE_VERSION=$(jq -r '.version' package.json)
+
+if [ "$PACKAGE_VERSION" != "$PUBLISHED_VERSION" ]; then
+  yarn publish || 0
+else
+  echo "The current version ($PACKAGE_VERSION) has not changed. If you would like to deploy a new" \
+    "version, follow the instructions in the readme."
```
Comment (jorge0136) [inline bin/publish]: I just looked at the readme. Do we still need to create a tagged commit? Should that happen in this script now?

File: bin/publish [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+# Add the NPM token to the .npmrc file
+echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> "$HOME/.npmrc"
+
+# Publish the package, but only if the current published version is behind the latest version.
+PUBLISHED_VERSION=$(yarn info --json | jq -r '.data."dist-tags".latest')
+PACKAGE_VERSION=$(jq -r '.version' package.json)
+
+if [ "$PACKAGE_VERSION" != "$PUBLISHED_VERSION" ]; then
+  yarn publish || 0
```
Comment (jorge0136) [inline bin/publish]: Do we want to have a `tag` or explicitly set this repo as `private`? Looking through the documentation for `yarn publish` I am not certain if we need anything else. https://yarnpkg.com/lang/en/docs/cli/publish/

Comment (LandonSchropp) [reply bin/publish]: Good question. When you run `yarn version`, it automatically generates a tag for you with the version number. The readme instructions show how to push tags as well.

File: bin/publish (hunk 2 above)
Comment (LandonSchropp) [reply bin/publish]: I don't think so. The main issues is there's no way for the script to know what version you'd like (major/minor/patch) in an automated way. I think we still have to do this step manually.

File: bin/publish (hunk 1 above)
Comment (LandonSchropp) [reply bin/publish]: Yep. It should always be there, and if it's not I'd definitely like to see the script fail.

**Pull request merged**